css-lists-3/Overview.bs

```diff
@@ -1142,7 +1142,7 @@ Instantiating Counters</h4>
 
 	When a [=counter=] is [=instantiated=] without an initial value,
 	the user agent must dynamically calculate the initial value at layout-time
-	to be the number of elements and pseudo-elements that [=instantiate=] the same counter in the same [=scope=], plus 1.
+	to be the number of elements and pseudo-elements that increment or set the same counter in the same [=scope=], plus 1.
```

Comment:
Or maybe just the number of list items (plus 1)?

Reply:
Yeah, that could work too but would make auto-reversed counters only work for list items, not for any custom counter. Though reversed counters on non-lists is probably a *very* rare use case.

Reply:
Oh, right